docs/pages/access-controls/access-request-plugins/ssh-approval-slack.mdx

```diff
@@ -32,13 +32,14 @@ Here is an example of sending an Access Request via Teleport's Slack plugin:
 - Slack admin privileges to create an app and install it to your workspace. Your
   Slack profile must have the "Workspace Owner" or "Workspace Admin" banner
   below your profile picture.
+- Either a Linux host or Kubernetes cluster where you will run the Mattermost plugin.
 
 (!/docs/pages/includes/tctl.mdx!)
 
 ## Step 1/8. Define RBAC resources
 
 Before you set up the Slack plugin, you will need to enable Role Access Requests
-in your Teleport cluster. 
+in the Proxy or Auth Service.
```

Comment:
```suggestion
in your Teleport cluster.
```
Not sure we need to be this specific in this intro sentence. We could even end the sentence at "enable Role Access Requests".